src/main/resources/changelogs/initialPopulateSlotsTable.xml

```diff
@@ -0,0 +1,91 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+         http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.0.xsd">
+
+    <property name="lastMonday" value="date_trunc('week', now())" dbms="postgresql"/>
```

Comment:
No, we won't. I'm gonna remove this autogenerating magic and hardcode those dates.